clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaConsumerService.java

```diff
@@ -360,6 +362,11 @@ public long getMaxElapsedTimeMSSinceLastPollInConsumerPool() {
             kafkaUrl,
             slowestTaskId,
             maxElapsedTimeSinceLastPollInConsumerPool);
+        LOGGER.info(
```

Comment:
Can we merge the extra info into the already-existing warn-level log, rather than add one more log statement?

Reply:
sure, will do